PhysicsPackages/eigenPhysicsPackage_class.f90

```diff
@@ -195,6 +195,8 @@ subroutine cycles(self, tally, tallyAtch, N_cycles)
         ! Obtain particle current cycle dungeon
         call self % thisCycle % copy(neutron, n)
 
+        neutron % showerID = n
```

Comment:
Could this line be included into `particleDungeon % copy` instead of here? I think it would look neater.

Reply:
Applied